odxtools/standardlengthtype.py

```diff
@@ -82,53 +78,132 @@ def __get_raw_mask(self, internal_value: AtomicOdxType) -> Optional[bytes]:
             # been anticipated by the standard, though...
             endianness = "little"
 
+        if self.is_condensed:
+            # if a condensed bitmask is specified, the number of bits
+            # set to one in the bit mask are used in the PDU
+
+            # TODO: this is pretty slow. replace it by
+            # `self.bit_mask.bit_count()` once we require python >=
+            # 3.10.
```

Comment:
we are not that far away from 3.9 end of life, you may drop it now

Reply:
I promised myself to support python 3.8 until ubuntu 20.04 is EOL. Fortunately, this is not too far off anymore (April), then we can do the same for Ubuntu 22.04 / python 3.10. I've added an item to the TODO list, though: https://github.com/mercedes-benz/odxtools/issues/286 . (That said, I suspect that the performance impact of converting the integer to a string and counting the one characters is lower than the interpreter overhead of doing two instead of one function calls. For compiled languages this is a completely different story...)